src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+            i && e(i).addClass("active");
+          }
+        }),
+        (n._slide = function (t, n) {
+          var i,
+            o,
+            r,
+            s = this,
+            a = this._element.querySelector(".active.carousel-item"),
+            c = this._getItemIndex(a),
+            u = n || (a && this._getItemByDirection(t, a)),
+            h = this._getItemIndex(u),
+            f = Boolean(this._interval);
+          if (
+            ("next" === t
+              ? ((i = "carousel-item-left"),
+                (o = "carousel-item-next"),
+                (r = "left"))
+              : ((i = "carousel-item-right"),
+                (o = "carousel-item-prev"),
+                (r = "right")),
+            u && e(u).hasClass("active"))
+          )
+            this._isSliding = !1;
+          else if (
+            !this._triggerSlideEvent(u, r).isDefaultPrevented() &&
+            a &&
+            u
+          ) {
+            (this._isSliding = !0),
+              f && this.pause(),
+              this._setActiveIndicatorElement(u);
+            var d = e.Event("slid.bs.carousel", {
+              relatedTarget: u,
+              direction: r,
+              from: c,
+              to: h,
+            });
+            if (e(this._element).hasClass("slide")) {
+              e(u).addClass(o), l.reflow(u), e(a).addClass(i), e(u).addClass(i);
+              var p = parseInt(u.getAttribute("data-interval"), 10);
+              p
+                ? ((this._config.defaultInterval =
+                    this._config.defaultInterval || this._config.interval),
+                  (this._config.interval = p))
+                : (this._config.interval =
+                    this._config.defaultInterval || this._config.interval);
+              var m = l.getTransitionDurationFromElement(a);
+              e(a)
+                .one(l.TRANSITION_END, function () {
+                  e(u)
+                    .removeClass(i + " " + o)
+                    .addClass("active"),
+                    e(a).removeClass("active " + o + " " + i),
+                    (s._isSliding = !1),
+                    setTimeout(function () {
+                      return e(s._element).trigger(d);
+                    }, 0);
+                })
+                .emulateTransitionEnd(m);
+            } else
+              e(a).removeClass("active"),
+                e(u).addClass("active"),
+                (this._isSliding = !1),
+                e(this._element).trigger(d);
+            f && this.cycle();
+          }
+        }),
+        (t._jQueryInterface = function (n) {
+          return this.each(function () {
+            var i = e(this).data("bs.carousel"),
+              o = s(s({}, v), e(this).data());
+            "object" == typeof n && (o = s(s({}, o), n));
+            var r = "string" == typeof n ? n : o.slide;
+            if (
+              (i || ((i = new t(this, o)), e(this).data("bs.carousel", i)),
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)